store/src/java/com/zimbra/cs/volume/VolumeServiceException.java

```diff
@@ -48,6 +49,17 @@ public final class VolumeServiceException extends ServiceException {
     public static final String BAD_VOLUME_STORE_MANAGER_CLASS               = "volume.STORE_MANAGER_CLASS";
     public static final String BAD_VOLUME_USE_IN_FREQUENT_ACCESS_THRESHOLD  = "volume.BAD_USE_IN_FREQUENT_ACCESS_THRESHOLD";
     public static final String BAD_VOLUME_GLOBAL_BUCKET_ID                  = "volume.BAD_GLOBAL_BUCKET_ID";
+    public static final String BAD_VOLUME_PROXY_PORT                        = "volume.BAD_PROXY_PORT";
+    public static final String BAD_VOLUME_ACCOUNT_PORT                      = "volume.BAD_ACCOUNT_PORT";
+    public static final String BAD_VOLUME_URL                               = "volume.BAD_URL";
```

Comment:
is admin console able to handle all these error cases?

Reply:
yes